### PR TITLE
feat(component): add unread message notification

### PR DESCRIPTION
**Pull request by fauzanlinnas**
> This project is a forked version of the [Sendbird UI Kit](https://github.com/sendbird/sendbird-uikit-react) to match with the Ruangguru's internal requirements. Therefore, it is not yet set up to accept pull requests from external contributors. But you can always freely create a forked version from this repository to match with your own requirements.

## Related Issue

[RKLS-936](https://ruanggguru.atlassian.net/browse/RKLS-936)

## Description Of Changes

- Change `Notification` component
- Add `Unread Notification` story

Screen Shot 2021-10-14 at 11 45 40

### CODE REVIEW

File: src/rogu/smart-components/Conversation/components/Notification.jsx [hunk 1]
```diff
@@ -13,11 +13,11 @@ export default function Notification({
 }) {
   const { stringSet } = useContext(LocalizationContext);
   const timeArray = time.split(' ');
-  timeArray.splice(-2, 0, stringSet.CHANNEL__MESSAGE_LIST__NOTIFICATION__ON);
+  timeArray.splice(1, 0, stringSet.CHANNEL__MESSAGE_LIST__NOTIFICATION__ON);
```
Comment (ibedwi) [inline src/rogu/smart-components/Conversation/components/Notification.jsx]: @fauzanlinnas kira-kira di bagian ini bisa ditambahin komentar ga ya? inputannya apa dan nanti hasilnya apa? kalau baca sekilas butuh trace lagi 😅

**Review by ibedwi — CHANGES_REQUESTED** [review]

**Comment by suliskh** [comment]
Have you checked the click-behaviour, Mas @fauzanlinnas? Based on the requirements document, it should be scrolled down to the latest message.

**Comment by fauzanlinnas** [comment]
> Have you checked the click-behaviour, Mas @fauzanlinnas? Based on the requirements document, it should be scrolled down to the latest message.

yes mas, itu `onClick`nya bakalan berasal dari Conversation/index.js (pas dipake). Di story sengaja dikosongin cuman buat nampilin UInya doang @suliskh

File: src/rogu/smart-components/Conversation/components/Notification.jsx (hunk 1 above)
Comment (fauzanlinnas) [reply src/rogu/smart-components/Conversation/components/Notification.jsx]: done mas

File: src/rogu/smart-components/Conversation/components/stories/index.stories.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React, { useRef, useState } from "react";
```
Comment (suliskh) [inline src/rogu/smart-components/Conversation/components/stories/index.stories.js]: Mas @fauzanlinnas, I think we can put this story inside the existing file (i.e. `.src/rogu/smart-components/Conversation/stories/index.stories.js`).

Also, please add the localization example in that story. You can refer to this: https://sendbird.com/docs/uikit/v1/react/guides/text-localization.

Thank you

**Review by suliskh — CHANGES_REQUESTED** [review]

File: src/rogu/smart-components/Conversation/components/stories/index.stories.js (hunk 2 above)
Comment (fauzanlinnas) [reply src/rogu/smart-components/Conversation/components/stories/index.stories.js]: mas, ku kurang paham maksud mas yang mesti di localization string yg mana ya?

Comment (suliskh) [reply src/rogu/smart-components/Conversation/components/stories/index.stories.js]: Maksudku ditambahkan contoh localization di story-nya, Mas @fauzanlinnas. Contohnya seperti di https://github.com/ruang-guru/sendbird-uikit-react/pull/16/files#diff-6f8ab15307cacad99d20adc6c92672595d56164d7fdae9cf8c3b01054c8d928d, aku nambahin contoh override `stringSet`-nya via `<SendbirdProvider stringSet={...}` />. Contoh overridingnya ada di `__mocks__/localizationMock`.

Comment (fauzanlinnas) [reply src/rogu/smart-components/Conversation/components/stories/index.stories.js]: done mas @suliskh  👍

Comment (suliskh) [reply src/rogu/smart-components/Conversation/components/stories/index.stories.js]: Awraitt, thank you mas

**Review by suliskh — APPROVED** [review]

**Review by ibedwi — APPROVED** [review]
LGTM!🚀

**Pull request merged**